jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/generator/QuarkusGeneratorTest.java

```diff
@@ -94,84 +99,42 @@ public void setUp() throws IOException {
   }
 
   @Test
-  public void isApplicable_withNoDependencies_shouldReturnFalse() {
+  void isApplicable_withNoDependencies_shouldReturnFalse() {
     // When
     final boolean result = new QuarkusGenerator(ctx).isApplicable(new ArrayList<>());
     // Then
     assertThat(result).isFalse();
   }
 
-  @Test
-  public void isApplicable_withQuarkusGroupIdPlugin_shouldReturnTrue() {
+  @ParameterizedTest(name = "{0}")
```

Comment:
Please, try to create a description using placeholders and the provided values, this one should be easy.